aroquev/src/main/webapp/style.css

```diff
@@ -8,7 +8,118 @@
   margin-top: 20px;
 }
 
-#main-photo {
+.body-img {
     width: 600px;
-    height: 400px;
+}
+
+/* Rules for the GitHub and LinkedIn icons */
+#github-link {
+    color: rgb(132, 0, 255);
+}
+
+#linkedin-link {
+    color: rgb(0, 81, 255);
+}
+
+#external-links img {
+    vertical-align: middle;
+    width: 50px;
+    height: 50px;
+}
+
+/* Rules for the photo gallery */
+* {box-sizing:border-box}
+
+/* Slideshow container */
+.slideshow-container {
+  max-width: 1000px;
+  position: relative;
+  margin: auto;
+}
+
+/* Hide the images by default */
+.mySlides {
+  display: none;
+}
+
+/* Next & previous buttons */
+.prev, .next {
+  cursor: pointer;
+  position: absolute;
+  top: 50%;
+  width: auto;
+  margin-top: -22px;
+  padding: 16px;
+  color: red;
+  font-weight: bold;
+  font-size: 30px;
+  transition: 0.6s ease;
+  border-radius: 0 3px 3px 0;
+  user-select: none;
+}
+
+/* Position the "next button" to the right */
+.next {
+  right: 0;
+  border-radius: 3px 0 0 3px;
+}
+
+/* On hover, add a black background color with a little bit see-through */
+.prev:hover, .next:hover {
+  background-color: rgba(0,0,0,0.8);
+}
+
+/* Caption text */
+.text {
+  color: #f2f2f2;
+  font-size: 15px;
+  padding: 8px 12px;
+  position: absolute;
+  bottom: 8px;
+  width: 100%;
+  text-align: center;
+  background-color: black;
+}
+
+/* Number text (1/3 etc) */
+.numbertext {
+  color: red;
+  font-size: 40px;
+  padding: 8px 12px;
+  position: absolute;
+  top: 0;
+}
+
+/* The dots/bullets/indicators */
+.dot {
+  cursor: pointer;
+  height: 15px;
+  width: 15px;
+  margin: 0 2px;
+  background-color: #bbb;
+  border-radius: 50%;
+  display: inline-block;
+  transition: background-color 0.6s ease;
```

Comment:
If you want to get really nerdy, you can look up the performance implications of animating different CSS properties at https://csstriggers.com/. Basically some properties are harder for browsers to animate than others. For example, you could make the browser do an ever-so-slightly less amount of work by animating `opacity` instead of `background-color`. Of course this will then blend the color of whatever the dot is on top of into the color of the dot, but will look almost the same.

As far as performance gains, the user will _never ever ever_ notice the difference with something this small, so it's not usually something worth worrying about. Larger animations that involve more expensive transitions might be noticed, but large animations are not really in-style these days.